packages/app-store/zoomvideo/lib/VideoApiAdapter.ts

```diff
@@ -59,6 +59,45 @@ export const zoomMeetingsSchema = z.object({
   ),
 });
 
+export type ZoomUserSettings = z.infer<typeof zoomUserSettingsSchema>;
+
+/** @link https://developers.zoom.us/docs/api/rest/reference/user/methods/#operation/userSettings */
+export const zoomUserSettingsSchema = z.object({
```

Comment:
I noticed we're only using the password and auto record settings. When making the API call can we filter for just those two properties.

Reply:
sure

Reply:
updated as per doc - https://developers.zoom.us/docs/api/rest/reference/user/methods/#operation/userSettings
<img width="1040" alt="Screenshot 2024-06-21 at 10 00 17 PM" src="https://github.com/calcom/cal.com/assets/3139634/0439e1a1-b69f-4e2f-94af-4f082471aebe">

Reply:
Awesome, can we also then clean up `zoomUserSettingsSchema` to only the fields we're filtering on. It'll help make reading the file easier.

Reply:
I can clean up.
I thought in future if any more settings is required to be retrieved , we already have the schema defined, need not again refer or search the zoom api doc.
Please share your thoughts.

Reply:
@joeauyeung , updated

Reply:
Our thought process is we only want code that is being used. If we need to access different user settings we can always add to the schema in the future.

Reply:
ok